webpack.AMDJqueryQ.config.js

```diff
@@ -24,7 +24,8 @@ module.exports = {
   },
   plugins: [
     new webpack.DefinePlugin({
-      SDK_VERSION: JSON.stringify(require('./package.json').version)
+      SDK_VERSION: JSON.stringify(require('./package.json').version),
+      STATE_TOKEN_COOKIE_NAME: JSON.stringify('oktaStateToken')
```

Comment:
Yeah, it's unusual. Webpack uses each of these global definitions as code fragments, so if you pass in a raw string, it just replaces the instances of the variable a raw string (no quotes). JSON.stringify adds quotes, so when it's pasted in the source, it's pasted with quotes. It's how it's described [in the docs](https://github.com/webpack/docs/wiki/list-of-plugins#defineplugin).

> If the value is a string it will be used as a code fragment.